### PR TITLE
Welcome book Rewrite and expansion.

### PR DESCRIPTION
**Pull request by RedDevel2**
My Rewrite for the Newfriend welcome book that has been in need for an update.

### CODE REVIEW

File: paper/config/plugins/SimpleAdminHacks/config.yml [hunk 1]
```diff
@@ -454,20 +454,29 @@ hacks:
...
-       - "If you would like to report hacks or glitches, please message the admins through subreddit modmail at reddit.com/r/CivMC\n\nDisclaimer: &2TangentialThreat&r is not staff and should not be trusted."
+       - "&5Hello and welcome to CivMC, &2${player}!\n\n   A server based on nation building and player freedom. Where admin control is minimal, and players dispense their own justice." 
+	   	   - "   &1Namelayer&r\nPocket radios! Create or join chat groups to communicate with friends all over CivMC. By default you have joined global chat or ! chat. You can switch from local chat (1000 block radius) to this one by performing the command: /g !"
+	   - "Go back to localchat by performing the /g command on it own. This works for any group you are allowed to speak in. just switch ! for the name of the group. \nCreate your own group using the GUI by perfoming the /nl Command."
+	  	   - "   &1Citadel&r\nAnother use for your Namelayer groups! Slow down raiders by reinforcing blocks such as your house or your chests and doors to lock them. Reinforced blocks take multiple breaks to drop for players not on the namelayer group."
```
Comment (dquist) [inline paper/config/plugins/SimpleAdminHacks/config.yml:462]: > not on the namelayer group


Can we call it 'reinforcement group' instead? IMO namelayer should just be the implementation detail for 'group management' which most players naturally understand.

File: paper/config/plugins/SimpleAdminHacks/config.yml [hunk 2]
```diff
@@ -454,20 +454,29 @@ hacks:
...
+	   - "To reinforce a block perform the command /ctr (Groupname) while holding stone, iron or diamond. then punch any block to reinforce it to that group. Now only your allies have ease of access! Tip: Use /ctf instead to reinforce while building!"
```
Comment (dquist) [inline paper/config/plugins/SimpleAdminHacks/config.yml:463]: - `(/ctr <groupname>)`
- Capitalize 'Then
- 'have ease of access' -> can interact with that block.

File: paper/config/plugins/SimpleAdminHacks/config.yml [hunk 3]
```diff
@@ -454,20 +454,29 @@ hacks:
...
+ 	   - "   &1Hiddenore&r\nSudden iron surrounds you! Ores spawn around you as you mine. While caves are still dusted with some visible ores, the main way to get ores is to dig. Ores still hold the same height level for their rarity.
```
Comment (dquist) [inline paper/config/plugins/SimpleAdminHacks/config.yml:464]: - 'Sudden' -> 'Suddenly,'

File: paper/config/plugins/SimpleAdminHacks/config.yml [hunk 4]
```diff
@@ -454,20 +454,29 @@ hacks:
...
+	   - "Diamonds spawn in veins, where you find one, you'll find others. A message will appear in chat when you've found one. Use silk touch on all ores for increased returns when smelting them in the Ore smelter factory. (better than fortune!)"
```
Comment (dquist) [inline paper/config/plugins/SimpleAdminHacks/config.yml:465]: We should add some clarity here as to what a vein means. As a new player, I wouldn't think of finding a single diamond block in isolation a 'vein'. We should describe how these veins are actually quite large in size, can contain hundreds of diamonds, but the ore blocks are separated by as many as 11 blocks so there is more digging required.

> Pro tip! Create waypoints using a client-side mod like JourneyMap when digging up ores in a vein to reveal the overall vein superstructure as you excavate it.

File: paper/config/plugins/SimpleAdminHacks/config.yml [hunk 5]
```diff
@@ -454,20 +454,29 @@ hacks:
...
+	   - "   &1JukeAlert&r\nSmile! you're on snitches! Jukeboxes and noteblocks when reinforced will turn into a snitch and notify players on the group when other players enter its 21x21 sphere. Noteblocks turned snitch only hold this one function." 
+	   - "jukeboxes in addition also log some actions, like block breaks or the opening of doors and chests. You can access a jukebox and it's logs if you're on its group by performing the /ja command. To name a nearby snitch perform the /janame (Snitchname)"
```
Comment (dquist) [inline paper/config/plugins/SimpleAdminHacks/config.yml:467]: - noteblock -> note-block
- 21x21 sphere -> 21x21 bounding box
- Capitalize Jukebox on L467.
- it's logs -> its logs
- `/janame Snitchname` -> `/janame <snitchname>`

File: paper/config/plugins/SimpleAdminHacks/config.yml [hunk 6]
```diff
@@ -454,20 +454,29 @@ hacks:
...
+ 	   - "	 &1ExilePearl&r\nKill other players with an ender pearl in your hotbar to banish them to the nether. A filled Pearl require essence to keep the player trapped. The location of the pearl is shown to the pearled player and can be broadcast to others. So watch your back..."
```
Comment (dquist) [inline paper/config/plugins/SimpleAdminHacks/config.yml:468]: Maybe reword so we don't encourage random killing of other players.

>Take justice into your own hands by killing another player with an enderpearl in your hotbar which traps them in the enderpearl. Players who are 'pearled' are trapped in the nether until they are released or until the upkeep essence runs out in its storage chest. The location of the pearl is shown to the pearled player and can be broadcast to others. So watch your back..."

**Review by dquist — COMMENTED** [review]
Overall - great changes! Left a few suggestions.

File: paper/config/plugins/SimpleAdminHacks/config.yml (hunk 5 above)
Comment (AngrySoundTech) [reply paper/config/plugins/SimpleAdminHacks/config.yml:467]: I can't remember which JA or Bastion measure from the center or don't include the center in the radius but I know they both do it differently, but I'm not sure which is 21x21 and which is 23x23

Comment (RedDevel2) [reply paper/config/plugins/SimpleAdminHacks/config.yml:467]: Please someone confirm what the actual size is. I always get them confused.

File: paper/config/plugins/SimpleAdminHacks/config.yml (hunk 6 above)
Comment (RedDevel2) [reply paper/config/plugins/SimpleAdminHacks/config.yml:468]: Yeah alright. Just changed "kill" to "killing" to make it sound less like an encouragement. 😄
